src/components/InventoryTabs/ConventionalSystems/Utilities.test.js

```diff
@@ -0,0 +1,14 @@
+import { calculateFilters } from './Utilities';
+
+describe('calculateFilters', () => {
+  it('should returns a mutated SearchParams for filters', () => {
+    const searchParams = new URLSearchParams(location.search);
+
+    expect(calculateFilters(searchParams, []).toString()).toEqual('');
+  });
+  it('should returns a mutated SearchParams for filters', () => {
+    const searchParams = new URLSearchParams(location.search);
+
+    expect(calculateFilters(searchParams, []).toString()).toEqual('');
```

Comment:
Looks duplicate to the previous test. Also, can we add some more scenarios with the second argument not being empty? Or perhaps let's leave a TODO note to work on that later if it feels overwhelming